src/Microsoft.DotNet.Tools.Publish/PublishCommand.cs

```diff
@@ -14,124 +14,74 @@
 
 namespace Microsoft.DotNet.Tools.Publish
 {
-    public class Program
+    public class PublishCommand
     {
-        public static int Main(string[] args)
-        {
-            DebugHelper.HandleDebugSwitch(ref args);
+        public string ProjectPath;
+        public string Configuration;
+        public string OutputPath;
+        public string Framework;
+        public string Runtime;
+        public NuGetFramework NugetFramework;
+        public IEnumerable<ProjectContext> ProjectRidFxPermutations;
```

Comment:
Agree here. Public fields are not good practice IMO

Java uses getters/setters to avoid them, C# has properties